en/06_Project_Governance/07_Supported_Modules.md

```diff
@@ -6,13 +6,20 @@ description: Modules which are commercially supported by Silverstripe
 # Commercially supported modules
 
 Silverstripe CMS ships modules that receive commercial support. Commercially supported modules receive regular updates to work with the latest Silverstripe CMS release. Their APIs conform to [semantic versioning](https://semver.org/). They are covered by:
+
 - our [security release process](/contributing/managing_security_issues) and
 - our [major release policy](major_release_policy).
 
 These modules will be supported for the lifetime of Silverstripe CMS 4 in the provided major versions.
 
 ## Core Silverstripe CMS modules
 
+The "core" of Silverstripe refers to the module packages owned by the "silverstripe" Packagist vendor that fall under one of the following recipes:
+
+- [silverstripe/recipe-core](https://packagist.org/packages/silverstripe/recipe-core)
+- [silverstripe/recipe-cms](https://packagist.org/packages/silverstripe/recipe-cms)
+- [silverstripe/installer](https://packagist.org/packages/silverstripe/installer)
```

Comment:
This was originally in the [maintainer guidelines](https://docs.silverstripe.org/en/project_governance/maintainer_guidelines/) page.